tutorials/get-started/index.md

```diff
@@ -0,0 +1,989 @@
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll learn:</b>
+
+* Basic structure of a Flutter app.
+* Finding and using packages to extend functionality.
+* Using hot reload for a quicker development cycle.
+* How to implement a stateful widget.
+* How to create an infinite, lazily loaded list.
+* How to create and navigate to a second screen.
+* .
+
+</div>
+
+<div class="whats-the-point" markdown="1">
+
+<b> <a id="whats-the-point" class="anchor" href="#whats-the-point" aria-hidden="true"><span class="octicon octicon-link"></span></a>What you'll use:</b>
+
+This tutorial requires version xxx or later of the Flutter SDK.
+When you install Flutter, you get the correct version of the
+Dart SDK as part of the install. You also get a Flutter Simulator,
+which you can use instead of a physical mobile device.
+
+These instructions feature the Android Studio IDE, but you can use whichever
+tools you prefer—Flutter and Dart plugins are also available for
+VSCode and JetBrains products.
+
+See [Mit's new setup page]() for information on how to set up
+your environment and create your first simple Flutter app.
+
+</div>
+
+## Step 1: Create the starting Flutter app
+
+Create a basic Flutter app, using the instructions in
+[Mit’s new page, preferably linked to a specific anchor for
+creating the templated app]().
+Name the project startup_namer. You’ll be modifying this starter
+app to create the finished app.
+
+### Update main.dart
+
+You'll update two files during this codelab. In the next step
+you'll update the pubspec file, `pubspec.yaml`, which manages
+the app's assets.
+You'll mostly be editing **lib/main.dart**, where the Dart code lives.
+
+<aside class="alert alert-success" markdown="1">
+<i class="fa fa-lightbulb-o"> </i> **Tip:**
+When pasting code into your app, indentation can
+become skewed. You can fix this in Android Studio by right-clicking the
+Dart code and selecting **Reformat Code with dartfmt**.
+Or, at the command line, you can use
+[dartfmt](https://github.com/dart-lang/dart_style).
+</aside>
+
+<ol markdown="1">
+
+<li markdown="1">
+Replace all of the code in lib/main.dart with the following,
+which displays "Hello World" in the center of the screen.
+
+<!-- skip -->
+{% prettify dart %}
+import 'package:flutter/material.dart';
+
+void main() => runApp(new MyApp());
+
+class MyApp extends StatelessWidget {
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Welcome to Flutter',
+      home: new Scaffold(
+        appBar: new AppBar(
+          title: new Text('Welcome to Flutter'),
+        ),
+        body: new Center(child: new Text('Hello World')),
+      ),
+    );
+  }
+}
+{% endprettify %}
+</li>
+
+<li markdown="1">
+Run the app. You should see the following screen.
```

Comment:
Here, and generally after each change, can we tell the developer to **hot reload** rather than **run**? Would need to check that all the changes actually do reload, though!

Reply:
OK, I just changed all instances to hot reload instead of run. But I need to go back and check because I think at least one of them requires a restart. But I noticed that AS TELLS you when you need to restart. Nice!